h/hoverIntent/hoverIntent_ubi8.5.sh

```diff
@@ -0,0 +1,45 @@
+# -----------------------------------------------------------------------------
+#
+# Package		: hoverIntent
+# Version		: v1.9.0
+# Source repo	: https://github.com/briancherne/jquery-hoverIntent
+# Tested on	: ubi 8.5
+# Script License: Apache License, Version 2 or later
+# Maintainer	: Sapna Shukla <Sapna.Shukla@ibm.com>
```

Comment:
Merging this, but please correct the formatting at later point and please avoid this in the future.